sys/include/net/nanocoap.h

```diff
@@ -285,6 +286,16 @@ typedef struct {
                                           1 for more blocks coming          */
 } coap_block1_t;
 
+/**
```

Comment:
Looking at use of this struct, would it be more descriptive to call it 'coap_blockbuilder_t' or something like that?

Reply:
Done